### DISCUSSION TITLE
Ensemble implementation

### DISCUSSION THREAD

**Discussion by jjakenichol** [jakobrunge/tigramite#164] [Q&A]
I just found this abstract: https://ui.adsabs.harvard.edu/abs/2021EGUGA..2310231R/abstract. I've been thinking about something that would take advantage of ensembles and I'm excited you all have worked on that already. Is there a public implementation or a paper out yet? I know you only just presented on it earlier this year.

Does running an ensemble help when a timeseries is relatively short? Say I have only 100 data points per variable. If I had say 10 samples of the same timeseries, would I get a good causal estimate? Have you looked at tradeoffs between sample size and number of samples?

What if timeseries in each sample were samples of the same causal structure (causal stationarity across samples), but different events occurred in each sample. This is something like having counterfactual samples. Can an ensemble method like yours learn from that? In my mind, it seems like that would be very useful but it's not clear if it would cause algorithmic challenges.

Thanks!

**Comment by jakobrunge** [comment]
This work is still under development, but a paper should come out in the summer!

**Comment by jakobrunge** [comment]
We have it now: https://github.com/jakobrunge/tigramite/blob/master/tutorials/dataset_challenges/tigramite_tutorial_multiple_datasets.ipynb

Even cooler, we have the following: https://github.com/jakobrunge/tigramite/blob/master/tutorials/causal_discovery/tigramite_tutorial_jpcmciplus.ipynb
